Frameworks/UIKit/NSString+UIKitAdditions.mm

```diff
@@ -334,7 +322,8 @@ - (CGSize)_sizeWithAttributes:(NSDictionary<NSString*, id>*)attributes constrain
 - (CGSize)sizeWithFont:(UIFont*)font constrainedToSize:(CGSize)size lineBreakMode:(UILineBreakMode)lineBreakMode {
     return [self _sizeWithAttributes:@{
         NSFontAttributeName : font,
-        NSParagraphStyleAttributeName : _paragraphStyleWithLineBreakMode(lineBreakMode)
+        static_cast<NSString*>(kCTParagraphStyleAttributeName) :
+            (id)[_paragraphStyleWithLineBreakMode(lineBreakMode) _createCTParagraphStyle]
```

Comment:
Design question: Why did we go for mirroring all the properties, instead of composing an NSParagraphStyle _from_ a CTParagraphStyle? That would have made this easier, made copy easier, and made mutable/immutable slightly easier.